db/migrate/20210614232549_add_index_smtp_group_id_email_logs.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddIndexSmtpGroupIdEmailLogs < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    execute <<~SQL
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS
```

Comment:
We're adding a null true column here so I don't think we need to add the index concurrently. Just add it when we add the column.